app/services/prison_visits/client.rb

```diff
@@ -52,11 +53,10 @@ def request(method, route, params)
         }
       }.deep_merge(params_options(method, params))
 
-      Rails.logger.info do
-        "Calling PVB API: #{method.to_s.upcase} #{path}"
-      end
-
-      response = @connection.request(options)
+      message = "Calling PVB API: #{api_method}"
+      response = Instrumentation.time_and_log(message, :api) {
```

Comment:
A single request can trigger more than one api request, how will the instrumentation code behave when that is the case? I'm worried that by using they same key (`api`) the previous values will get overwritten.